FFMpeg decoder rocdecode integration (#583)
ROCm/rocDecode: CHANGELOG.md

* HEVC stream bit depth change handling and DPB buffer size change handling through decoder reconfiguration.
* AVC stream DPB buffer size change handling through decoder reconfiguration.
* rocDecode now uses the Cmake CMAKE_PREFIX_PATH directive.

> * rocDecode - Add new avcodec based decoder which is built as a seperate "rocdecode-host" library

Reviewer: ```suggestion
* A new avcodec-based decoder built as a seperate "rocdecode-host" library has been added.
```

Does this need to be added to any part of the rocDecode documentation?
